airflow/serialization/serialized_objects.py

```diff
@@ -629,7 +629,7 @@ def _serialize_node(cls, op: Union[BaseOperator, MappedOperator], include_deps:
         serialize_op['_task_module'] = getattr(op, "_task_module", type(op).__module__)
 
         # Used to determine if an Operator is inherited from DummyOperator
-        serialize_op['_is_dummy'] = op.inherits_from_dummy_operator
+        serialize_op['_is_empty'] = op.inherits_from_empty_operator
```

Comment:
You should make a change in the deserialize code path so that if it sees `_is_dummy` it renames it to `_is_empty` -- although the DAG will get re-serialized eventually it's much nicer for bigger deployments if we can just seamless load the older serialization format.

Reply:
Good point.
Is https://github.com/apache/airflow/pull/22832/commits/3aacf0a1b54214291ef6551bfb48d9e46e3c77f9 is what you had in mind?

Reply:
Done.
also added a test https://github.com/apache/airflow/pull/22832/commits/fb87d0c63aadbf139a5bee353f4db290c5cc6794